include/bluetooth/direction.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+
+/** @brief Constant Tone Extension parameters for connectionless
+ * transmission.
+ *
+ * The structure holds information required to setup CTE transmission
+ * in periodic advertising.
+ */
+struct bt_le_df_adv_cte_tx_params {
+	/* Length of CTE in 8us units */
+	uint8_t  cte_len;
+	/* CTE Type: AoA, AoD 1us slots, AoD 2us slots */
+	uint8_t  cte_type;
+	/* Number of CTE to transmit in each periodic adv interval */
+	uint8_t  cte_count;
+	/* Number of Antenna IDs in the switch pattern */
+	uint8_t  num_ant_ids;
+	/* List of antenna IDs in the pattern */
+	uint8_t  *ant_ids;
+};
+
+/**
```

Comment:
Please start the @brief on the first line

Reply:
corrected